app/models/routing/routing_tag.rb

```diff
@@ -32,8 +32,13 @@ class Routing::RoutingTag < ApplicationRecord
            class_name: 'RateManagement::Project',
            autosave: false
 
-  has_many :rate_management_pricelist_items,
-           ->(tag) { unscope(:where).where("? = ANY(#{table_name}.routing_tag_ids)", tag.id) },
+  has_many :active_rate_management_pricelist_items,
+           ->(tag) { unscope(:where).not_applied.where("? = ANY(#{table_name}.routing_tag_ids)", tag.id) },
+           class_name: 'RateManagement::PricelistItem',
+           autosave: false
+
+  has_many :applied_rate_management_pricelist_items,
+           ->(tag) { unscope(:where).applied.where("? = ANY(#{table_name}.routing_tag_ids)", tag.id) },
            class_name: 'RateManagement::PricelistItem',
            autosave: false
```

Comment:
`applied_rate_management_pricelist_items` is not used anywhere so can be removed